src/main/java/org/catrobat/jira/timesheet/activeobjects/impl/TimesheetImpl.java

```diff
@@ -41,4 +41,23 @@ public TimesheetEntry firstEntry() {
 
         return entries.get(0);
     }
+
+    public final int calculateTotalHours()
+    {
+        TimesheetEntry[] TimeSheetEntryiesArray = timesheet.getEntries();
+
+        int sumTotalMinutes = 0;
+
+        if (TimeSheetEntryiesArray == null || TimeSheetEntryiesArray.length == 0) {
+            return 0;
+        }
+
+        for (TimesheetEntry entry : TimeSheetEntryiesArray)
```

Comment:
Please use uniform coding style (no new line for curly brackets).

Reply:
corrected brackets